app/views/shared/_personal_key_input.html.erb

```diff
@@ -6,6 +6,15 @@
        pattern="^<%= PersonalKeyFormatter.regexp_string %>$"
        required=""
        aria-invalid="false"
+       aria-describedby="personal-key-validation-message"
```

Comment:
Should this only be added if `validation_needed` is `true`, so that it's not referencing an element that doesn't exist otherwise?